music21/metadata/__init__.py

```diff
@@ -807,17 +1325,12 @@ def composers(self) -> t.List[str]:
         >>> md.composers
         []
         '''
-        return self._contributor_multiple_role_getter('composer')
-
-    @composers.setter
-    def composers(self, value: t.List[str]) -> None:
-        self._contributor_multiple_role_setter('composer', value)
-
+        return self._getPluralAttribute('composer')
 
     @property
-    def date(self):
+    def dateCreated(self):
         r'''
-        Get or set the date of this work as one of the following date
+        Get or set the creation date of this work as one of the following date
         objects:
```

Comment:
-1.  disagree with the rename without a deprecation cycle for date.  We should still allow date as a synonym for dateCreated.  It is used extensively in other docs and will break parsing of custom corpora.

(Speaking of custom corpora -- have you tried doing a corpus.search on one generated in the old metadata format to ensure that it gives a meaningful error message?  Ideally it would reindex automatically, but fine would be to detect old format and give an explanation of how to reindex).

Reply:
Yes on the deprecation.  I wouldn't know how to run that corpus.search test.

Reply:
Deprecation is done.

Reply:
Making an issue to test.  Custom corpora aren't so often used, but they are used by some of music21's most demanding and loyal users.